frontend/src/components/pages/RegistrantExperience/RegistrationSteps/RegistrationFooter.tsx

```diff
@@ -39,16 +41,19 @@ const RegistrationFooter = ({
       bottom="0"
       zIndex="5"
     >
-      <Button
-        width={{ sm: "95vw", md: "45vw", lg: "auto" }}
-        height="48px"
-        variant="secondary"
-        onClick={() => handleStepNavigation(-1)}
-        mb={{ sm: 4, md: 0 }}
-        mr={{ sm: 0, md: 4 }}
-      >
-        Back
-      </Button>
+      {(!isWaitlistRegistration ||
+        currentStep !== RegistrantExperienceSteps.PersonalInfoPage) && (
```

Comment:
This logic is somewhat difficult to parse, can we write this as `!(isWaitListRegistration && currentStep === Reg...PersonalInfoPage)` maybe?